packages/terra-clinical-result/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed the spacing between the date in time for the ResultTimeHeaderCell.
```

Comment:
nit: `'date and time in the ResultTimeHeaderCell component.'`

Reply:
good catch https://github.com/cerner/terra-clinical/pull/876/commits/8c82d1fba17437653c3577a8f926b5b8a0176e0e